src/main/java/sx/blah/discord/handle/impl/obj/Guild.java

```diff
@@ -779,6 +778,18 @@ public IEmoji getEmojiByName(String name) {
 	}
 
 	@Override
+	public IEmoji addEmoji(String name, BufferedImage image){
```

Comment:
We talked about this a while ago in the api-help channel on discord and determined that Buffered Image yas the best way because it was the only way to convert to byte[]